### ISSUE TITLE
Metamask is not able to execute normal transactions in private quorum blockchain

### ISSUE THREAD

**Issue by Tsajan** [Consensys-Incorporated/quorum#1021]
Hi!

I am setting up a quorum network with two nodes from the scratch which are using the RAFT consensus mechanism. The two nodes are live, up and running. One is specified as minter and the other is verifier. I have deployed a smart contract into the private quorum chain network using the Truffle framework. Next, I would like to access the accounts in web UI. So, I connected Metamask wallet to the quorum network. I was able to add the contract token and account info to Metamask and retrieve corresponding ETH balances and tokens balances. However, sending a normal transaction (transfer ether between two addresses) using Metamask results in the following error

```
MetaMask - RPC Error: Error: [ethjs-rpc] rpc error with payload {"id":6378119053557,"jsonrpc":"2.0","params":["0xf88b17850af16b1600830f424094e8bd1fc300c3cd85bf033a13effe02226d22e76280a4c6ed8990000000000000000000000000000000000000000000000000000000000000000a82f4f5a04e31045bf76c16a594ee742be05909019bfe31b0c57cca66918b13898b3684fda023aa10926436f4eec79d2a08407a55ba35d1dad4d50d9029dc26d7d7629bfabf"],"method":"eth_sendRawTransaction"} [object Object]
```
How can I overcome this issue while using Metamask

**Comment by Tsajan** [comment]
I am currently using Metamask version 7.7.9

**Comment by fixanoid** [comment]
Did you set the price of gas to 0 when sending this txn?

**Comment by fixanoid** [comment]
Also, while this is a bit dated, it does cover MetaMask still: https://docs.google.com/presentation/d/1vMBVKTJyKbGwZaMqwxA7VaCTEXELA5kLf56byZIbD3g/edit#slide=id.g54ef6f9849_1_20

Start from page 25.

**Comment by Tsajan** [comment]
Thank you for the quick response, but still it is not being solved. I would like to send transactions between two accounts in quorum network using metamask. I have added a network connection in metamask linking to my quorum network as-
![image](https://user-images.githubusercontent.com/7326736/85419481-1b499c00-b5ad-11ea-8eb2-8f1430a2cc1c.png)

And I would like to send some ether to another account in the same network using Metamask, so I did like this
![image](https://user-images.githubusercontent.com/7326736/85419643-4df39480-b5ad-11ea-9bac-8e8ef6244593.png)

Like you suggested, I modified the gas price to 0 and proceeded to submit the transaction.
![image](https://user-images.githubusercontent.com/7326736/85419753-6c599000-b5ad-11ea-844d-8fad2c7e5646.png)

This results in the error
![image](https://user-images.githubusercontent.com/7326736/85419865-85624100-b5ad-11ea-8a0e-e1c6e52886b0.png)

**Comment by Tsajan** [comment]
Sorry, but this is not yet fixed. So please don't close this

**Comment by fixanoid** [comment]
@Tsajan we commonly close issues that do not originate from Quorum, I will still assist on closed tickets tho.

Please check what the node log is reporting for this issue. Occasionally, if a txn was sent from an account or with options that Quorum cannot process, it will be stuck as pending. If this is the case, you may need to reset the txnpool on the node you are connecting to. Alternatively, try creating a new account and sending txn from that.

If you'd like an immediate response so we could assist with debugging, please join our slack https://www.goquorum.com/slack-inviter

**Comment by Tsajan** [comment]
I am able to successfully send transaction at geth level by using web3 rpc calls. I have used the following geth commands to send balance from the coinbase accounts to Account 1 listed above. Let me send 10 ethers from coinbase to Account 1. I have used the following geth commands

```
eth.sendTransaction({from:eth.accounts[0], to:"0x47A1e3FCE50190245f1Ae9943D3E59e91B9C11Ef", value:10000000000000000000})

```
![image](https://user-images.githubusercontent.com/7326736/85424556-0a038e00-b5b3-11ea-9e98-427dd14fc84e.png)

I can confirm the update in balance in Metamask extension as-
![image](https://user-images.githubusercontent.com/7326736/85424672-2bfd1080-b5b3-11ea-8fe8-a179ac791395.png)
